contrib/completion/bash/docker

```diff
@@ -447,17 +462,20 @@ _docker_images() {
 			COMPREPLY=( $( compgen -W "true false" -- "${cur#=}" ) )
 			return
 			;;
+		*label=*)
+			return
+			;;
```

Comment:
You're right, there is an asymmetry in this regard between the completions for `docker ps` and `docker images`. It is intended, however.
The point is that `docker images` takes an optional image specification argument. Without the `*label=*` expression, `-f label=` would match `*)` and thus complete images instead of "anything".
As `docker ps` has no such argument, there's no need to deal with a catch-all glob.